src/pages/signin/LoginForm.js

```diff
@@ -1,25 +1,29 @@
 import React from 'react';
 import {TextInput, View} from 'react-native';
-import {withOnyx} from 'react-native-onyx';
+import Onyx, {withOnyx} from 'react-native-onyx';
 import PropTypes from 'prop-types';
 import _ from 'underscore';
 import styles from '../../styles/styles';
 import themeColors from '../../styles/themes/default';
 import Button from '../../components/Button';
 import Text from '../../components/Text';
-import {fetchAccountDetails} from '../../libs/actions/Session';
 import ONYXKEYS from '../../ONYXKEYS';
 import withWindowDimensions, {windowDimensionsPropTypes} from '../../components/withWindowDimensions';
 import compose from '../../libs/compose';
 import canFocusInputOnScreenFocus from '../../libs/canFocusInputOnScreenFocus';
 import withLocalize, {withLocalizePropTypes} from '../../components/withLocalize';
 import getEmailKeyboardType from '../../libs/getEmailKeyboardType';
+import {fetchAccountDetails} from '../../libs/actions/Session';
 
 const propTypes = {
     /* Onyx Props */
 
     /** The details about the account that the user is signing in with */
     account: PropTypes.shape({
+
+        /** Login of the account */
+        login: PropTypes.string,
```

Comment:
I'm not sure if it's the best idea to use the `account.login` key for this. Probably it would be safer to create a new Onyx key to share the state between the different login forms. A lot of things are referencing `account.login` and we are kind of using it for two purposes now (which I find confusing).

Reply:
>  A lot of things are referencing account.login and we are kind of using it for two purposes now (which I find confusing)

I think it's `credentials.login` that's being used everywhere not `account.login`. 
But yes, we can use a different key for this

https://github.com/Expensify/App/blob/21b1ee466b7084b3d6d11602d6b09e368ab02f88/src/ONYXKEYS.js#L5-L6

 https://github.com/Expensify/App/blob/21b1ee466b7084b3d6d11602d6b09e368ab02f88/src/ONYXKEYS.js#L33-L34

Reply:
Ah, sorry, you are right and I am wrong :) Thanks! This seems fine then.